src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
@MoOx  @tabatkins does the nesting spec takes in consideration `custom selectors` I think that is the concern here!